trueline.sh

```diff
@@ -407,6 +425,7 @@ if [[ "${#TRUELINE_SEGMENTS[@]}" -eq 0 ]]; then
     declare -a TRUELINE_SEGMENTS=(
         'user,black,white,bold'
         'venv,black,purple,bold'
+	    # 'conda_env,black,soft_red,bold'
```

Comment:
Keep indentation exactly the same as the commented line below (the one about the newline segment)

Reply:
I assume the indentation is same as the one for newline
![rsz_screenshot_from_2020-08-02_06-29-48](https://user-images.githubusercontent.com/18904157/89113082-ce00fa00-d489-11ea-9490-396e56ae8116.png)

Reply:
there seems to be some issues in GitHub PR view it's fine in my local machine as well as in my fork